scene/resources/style_box.cpp

```diff
@@ -618,17 +630,28 @@ inline void draw_ring(Vector<Vector2> &verts, Vector<int> &indices, Vector<Color
 		}
 	}
 
-	int vert_count = (adapted_corner_detail + 1) * 4 * 2;
+	int vert_count = verts.size() - vert_offset;
+
 	//fill the indices and the colors for the border
 	for (int i = 0; i < vert_count; i++) {
-		//poly 1
 		indices.push_back(vert_offset + ((i + 0) % vert_count));
```

Comment:
here you are totally right, that everything gets drawn twice. I would have needed to do:
`int vert_count = (adapted_corner_detail + 1) * 4;` without the `* 2`.

As I explained in the post before, the issue with the bright line is still caused by the Anti aliasing. This effect causes a bug, when using the same (semi transparent) color for the border and the infill. Than the border is drawn twice -> is darker.

Reply:
Here is a screenshot:
<img width="327" alt="Screen Shot 2019-03-17 at 19 16 08" src="https://user-images.githubusercontent.com/16718859/54495947-8f48bf00-48e9-11e9-8e36-53bbc1eb25e9.png">